mappings/servers/hvbc/metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+    "id": "hvbc",
+    "name": "hvbc",
```

Comment:
Please use the actual server name here.
I'm guessing that it should be `HVBC` in your case (capitalized)
Unless you have a full server name